app/src/main/res/values-night/themes_material3.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright © 2014-2021 The Android Password Store Authors. All Rights Reserved.
+  ~ SPDX-License-Identifier: GPL-3.0-only
+  -->
+
+<resources>
+
+  <style name="APSThemeM3" parent="Theme.Material3.DayNight">
+    <item name="colorPrimary">@color/md_theme_dark_primary</item>
+    <item name="colorOnPrimary">@color/md_theme_dark_onPrimary</item>
+    <item name="colorPrimaryContainer">@color/md_theme_dark_primaryContainer</item>
+    <item name="colorOnPrimaryContainer">@color/md_theme_dark_onPrimaryContainer</item>
+    <item name="colorSecondary">@color/md_theme_dark_secondary</item>
+    <item name="colorOnSecondary">@color/md_theme_dark_onSecondary</item>
+    <item name="colorSecondaryContainer">@color/md_theme_dark_secondaryContainer</item>
+    <item name="colorOnSecondaryContainer">@color/md_theme_dark_onSecondaryContainer</item>
+    <item name="colorTertiary">@color/md_theme_dark_tertiary</item>
+    <item name="colorOnTertiary">@color/md_theme_dark_onTertiary</item>
+    <item name="colorTertiaryContainer">@color/md_theme_dark_tertiaryContainer</item>
+    <item name="colorOnTertiaryContainer">@color/md_theme_dark_onTertiaryContainer</item>
+    <item name="colorError">@color/md_theme_dark_error</item>
+    <item name="colorErrorContainer">@color/md_theme_dark_errorContainer</item>
+    <item name="colorOnError">@color/md_theme_dark_onError</item>
+    <item name="colorOnErrorContainer">@color/md_theme_dark_onErrorContainer</item>
+    <item name="android:colorBackground">@color/md_theme_dark_background</item>
+    <item name="colorOnBackground">@color/md_theme_dark_onBackground</item>
+    <item name="colorSurface">@color/md_theme_dark_surface</item>
+    <item name="colorOnSurface">@color/md_theme_dark_onSurface</item>
+    <item name="colorSurfaceVariant">@color/md_theme_dark_surfaceVariant</item>
+    <item name="colorOnSurfaceVariant">@color/md_theme_dark_onSurfaceVariant</item>
+    <item name="colorOutline">@color/md_theme_dark_outline</item>
+    <item name="colorOnSurfaceInverse">@color/md_theme_dark_inverseOnSurface</item>
+    <item name="colorSurfaceInverse">@color/md_theme_dark_inverseSurface</item>
```

Comment:
nit: can we rename these to onSurfaceInverse and surfaceInverse to follow the pattern?